.gitignore

```diff
@@ -67,7 +67,7 @@ dist/
 !docs/website/*
 !docs/website/dist/*
 !docs/website/node_modules/*
-sandbox/
+#sandbox/
```

Comment:
Keep sandbox folder removed from commits to avoid unnecessary work to resolve commits from this file.